docs/configuration.md

```diff
@@ -0,0 +1,696 @@
+            useHDAddresses: True
+            totalBalance: 600000000000000000
+          fakeAvvmBalance:
+            count: 10
+            oneBalance: 100000
+          avvmBalanceFactor: 0.5
+          useHeavyDlg: True
+          seed: 0
+        blockVersionData:
+          scriptVersion:     0
+          slotDuration:      7000
+          maxBlockSize:      2000000
+          maxHeaderSize:     2000000
+          maxTxSize:         4096 # 4 Kb
+          maxProposalSize:   700 # 700 bytes
+          mpcThd:            0.01 # 1% of stake
+          heavyDelThd:       0.005 # 0.5% of stake
+          updateVoteThd:     0.001 # 0.1% of total stake
+          updateProposalThd: 0.1 # 10% of total stake
+          updateImplicit:    10 # slots
+          softforkRule:
+            initThd:        0.9 # 90% of total stake
+            minThd:         0.6 # 60% of total stake
+            thdDecrement:   0.05 # 5% of total stake
+          txFeePolicy:
+            txSizeLinear:
+              a: 155381 # absolute minimal fees per transaction
+              b: 43.946 # additional minimal fees per byte of transaction size
+          unlockStakeEpoch: 18446744073709551615 # last epoch (maxBound @Word64)
+        protocolConstants:
+          k: 2
+          protocolMagic: 55550001
+          vssMinTTL: 2
+          vssMaxTTL: 6
+        ftsSeed: "c2tvdm9yb2RhIEdndXJkYSBib3JvZGEgcHJvdm9kYSA="
+        heavyDelegation: {}
+        avvmDistr:
+          5ZI8dniHQgyz1N/M6wQa4UDe5imEg/83a1IFmlkP6PI=: 101
+```
+
+Most values are the same as in _genesis data_. It's easier to write about
+differences:
+1. Genesis spec has `initializer` value not present in genesis
+   data. It's described in more details [below](#initializer).
+2. Thesholds are specified as floating-point numbers.
+3. `ftsSeed` is base64-encoded, not hex-encoded.
+4. `txFeePolicy` is specified slightly differently. The comments in
+   the example explain the format pretty well. Note that the values
+   are lovelaces, not ADA.
+5. Addresses in `avvmDistr` are presented in base64 format, not base64(url).
+
+#### Initializer
+
+Sample configuration with `initializer` was presented above. It
+contains the following fields:
+* `testBalance`. It specifies how many nodes should be rich, how many
+  nodes should be poor, how big should be total balance of rich nodes,
+  total balance and whether to use HD addresses or simple PubKey
+  addresses. Assignment of balances and stakes is described in
+  detail [below](#balances-and-stakes).
+* `fakeAvvmBalance`. It contains number of fake AVVM seeds and how
+  balance of each seed. These seeds can be used to redeem ADA.
+* `avvmBalanceFactor`. Real AVVM balances will be multiplied by this
+  factor. Fake AVVM balances are not affected.
+* `useHeavyDlg`. If it's `True`, then for each bootstrap stakeholders
+  there will be a heavyweight delegation certificate from this
+  stakeholder to a core node. Otherwise bootstrap stakeholders will
```

Comment:
"core node" here is ambiguous and it wasn't defined previously in the document. I suggest to use "rich node" instead.